integration/standalone_tar_test.go

```diff
@@ -281,19 +279,6 @@ func TestTar(t *testing.T) {
 				osArgs = append(osArgs, "--input", tt.testArgs.Input)
 			}
 
-			// TODO: test skip files/dirs
-			if len(tt.testArgs.SkipFiles) != 0 {
```

Comment:
Could you keep them as is? The tests you added are for filesystem scanning and this is for image scanning.

Reply:
Add tests https://github.com/aquasecurity/trivy/pull/2984/commits/ce3dcb6d927db4dfc0bd7d9a3b7c526ccbd6eb0e